README.md

```diff
@@ -1,55 +1,166 @@
+
 # SwitchbOTA
-Replaces the default firmware on the SwitchBot Plug Mini via OTA, enabling the use of Tasmota without disassembling the unit.
+![51HjLj15OZL _SL1500_](https://github.com/kubedzero/switchbota/assets/51148401/0bdf7ae9-9eaf-4698-819e-78625adce8fc)
```

Comment:
can you move these images into the repo + adjust the urls rather than link against gh assets